drivers/state.go

```diff
@@ -124,52 +124,63 @@ func (d *StateDriver) newMinerAccountActor() address.Address {
 	expectedMinerActorIDAddress := utils.NewIDAddr(d.tb, utils.IdFromAddress(minerWorkerID)+1)
 	minerActorAddrs := computeInitActorExecReturn(d.tb, minerWorkerPk, 0, 1, expectedMinerActorIDAddress)
 
-	// create the miner actor so it exists in the init actors map
+	// create the miner actor s.t. it exists in the init actors map
 	_, minerActorIDAddr, err := d.State().CreateActor(builtin_spec.StorageMinerActorCodeID, minerActorAddrs.RobustAddress, big_spec.Zero(), &miner_spec.State{
-		PreCommittedSectors: EmptyMapCid,
-		Sectors:             EmptyArrayCid,
-		FaultSet:            abi_spec.NewBitField(),
-		ProvingSet:          EmptyArrayCid,
 		Info: miner_spec.MinerInfo{
 			Owner:            minerOwnerID,
 			Worker:           minerWorkerID,
 			PendingWorkerKey: nil,
 			PeerId:           "chain-validation",
 			SectorSize:       0,
```

Comment:
This value isn't valid. I suggest using 2048 if you need a constant (define this in a constant somewhere)